packages/terra-navigation-prompt/src/NavigationPromptCheckpoint.jsx

```diff
@@ -149,29 +149,31 @@ class NavigationPromptCheckpoint extends React.Component {
       title, message, acceptButtonText, rejectButtonText, emphasizedAction, onAccept, onReject,
     } = this.state.notificationDialogProps;
 
+    const acceptActionIsEmphasized = !emphasizedAction || emphasizedAction === 'accept';
+
     const acceptButton = {
       text: acceptButtonText,
       onClick: () => {
         this.setState({ notificationDialogProps: undefined }, onAccept);
       },
+      isEmphasized: (acceptActionIsEmphasized),
     };
 
     const rejectButton = {
       text: rejectButtonText,
       onClick: () => {
         this.setState({ notificationDialogProps: undefined }, onReject);
       },
+      isEmphasized: (!acceptActionIsEmphasized),
     };
 
-    const acceptActionIsEmphasized = !emphasizedAction || emphasizedAction === 'accept';
-
     return (
       <NotificationDialog
         isOpen
         title={title}
-        message={message}
-        primaryAction={acceptActionIsEmphasized ? acceptButton : rejectButton}
-        secondaryAction={acceptActionIsEmphasized ? rejectButton : acceptButton}
+        startMessage={message}
+        acceptAction={acceptActionIsEmphasized ? acceptButton : rejectButton}
+        rejectAction={acceptActionIsEmphasized ? rejectButton : acceptButton}
```

Comment:
why are we using emphasis to switch button order? what about the utilizing the isReversed prop that's been added?

Reply:
Originally I was trying to make the changes as passive as possible, upon further discussion with Neil I have updated NavigationPromptCheckpoint to follow the same prop logic we use in NotificationDialog. https://github.com/cerner/terra-framework/pull/899/commits/67555271fd37cfa811ddb2cf31151f65149929b1